metrics.go

```diff
@@ -21,21 +21,38 @@ var (
 
 func sanitizeValue(label string, value interface{}) string {
 
-	// Convert to a string, no matter what underlying type it is
-	var labelValue string
-	if value != nil {
-		labelValue = fmt.Sprintf("%v", value)
+	if value == nil || value == "" || value == "-" {
+		return ""
 	}
 
+	// Convert to a string, no matter what underlying type it is
+	labelValue := fmt.Sprintf("%v", value)
+	labelValue = strings.TrimSpace(labelValue)
+
 	switch label {
 	case "content_type":
```

Comment:
In logstash we kept the buckets aligned with the format of the content-type header:
https://github.com/section-io/section.aperture/blob/master/puppet/squixa_v3_logs_indexer/files/etc-logstash-processor-b-conf.d/10-shared-ingress.conf#L250-L277
Consider same, or maybe the label should not be called `content_type` but instead `content_type_bucket`?